components/admin_console/schema_admin_settings.jsx

```diff
@@ -1064,6 +1064,7 @@ export default class SchemaAdminSettings extends React.PureComponent {
                     />
                     <div
                         className='error-message'
+                        data-testid='error-message'
```

Comment:
nit: If possible, please change to `errorMessage` for consistency per [E2E docs](https://developers.mattermost.com/contribute/webapp/end-to-end-tests/#writing-specs), although (sadly) we're not consistent in other places of the codebase.